testcode/lib/conio.c

```diff
@@ -120,6 +131,10 @@ void main(void)
                     cputc(i);
                     inpos = (inpos + 1) & 7;
                 }
+#endif
+#if defined(__NES__) || defined(__PCE__) || defined(__GAMATE__)
+                waitvblank();
```

Comment:
"something" must limit the speed the inner loop is running at, else the revers printed char(s) do not blink but just flicker irritatingly fast :)